<launch>
    <node name="human_follower_node" pkg="human_follower" type="human_follower_node" output="screen">
        <!-- ParÃ¡metros para el nodo -->
        <param name="dist_to_human" value="1.0"/>
        <param name="max_linear" value="0.6"/>
        <param name="move_head" value="true"/>
        <param name="pot_fields" value="false"/>
        <param name="legs_pose_topic" value="/hri/leg_finder/leg_pose"/>
        <param name="cmd_vel_topic" value="/hsrb/command_velocity"/>
        <param name="head_topic" value="/hardware/head/goal_pose"/>
        <param name="enable_potfields" value="true"/>
    </node>
    <node name="leg_finder_node" pkg="leg_finder" type="leg_finder_node" output="screen"></node>
</launch>
